tweeter/target/classes/twitter4j.properties

```diff
@@ -0,0 +1,9 @@
+//debug=true
```

Comment:
These type of configuration files and credentials should NEVER EVER be added to the repository, especially repositories of this nature, now Anybody can post anything using your credentials.